### ISSUE TITLE
Display errors occurred during validation of the service

### ISSUE THREAD

**Issue by judepereira** [storyscript/hub#99]
What is one supposed to do after seeing this error?

![Screenshot 2019-08-20 at 14 46 55](https://user-images.githubusercontent.com/747087/63348410-74c3b780-c359-11e9-9f46-77d9493f6562.png)

**Comment by JeanBarriere** [comment]
that means the microservice.yml does not pass the omg validation. So you should run omg validate on your microservice.yml to see what you did wrong

**Comment by judepereira** [comment]
Yes, duly noted. I did that, but the experience would have been better if the errors were shown upfront. Also, we're missing a button there to check the updated copy again. One needs to go back and start the whole process to see if it's valid or not.

**Comment by JeanBarriere** [comment]
For now, you just have to click on whether previous or the "select a repository" button.
But I definitely agree this needs improvements, let's have all of these feedbacks given to Ayda when she'll be on this part

**Comment by judepereira** [comment]
@rohit121 Please note this improvement for the Hub.

**Comment by rohit121** [comment]
@judepereira  thanks
